project/CodeStone1125/README.md

```diff
@@ -0,0 +1,149 @@
+# 2D Collision Detector
+
+
+## Basic Information
+
+Simulate how different type of shape collide in a 2D map.
+
+* github repository: [https://github.com/CodeStone1125/2DCollisionDetector](https://github.com/CodeStone1125/2DCollisionDetector)
+
+## Problem to Solve
+
+The problem this system wants to solve is to calculate situation for the most common shapes in 2D map like retangle, circle and Polygon.
+
+There are some algorithms can be implemented for solving this problem. Before we dive into the topic deeper, let's first talk about the the "Axis-Aligned Bounding Box" (AABB) algorithm. While it's not a single algorithm but a framework for collision detection, it forms the basis for many collision detection systems in 2D graphics and game development due to its simplicity and efficiency.
```

Comment:
Using a single line for a paragraph makes it hard to add comments for the "code".  The plain text in the proposal is treated as "code".  The code in programming language should use proper line width.  So is the "code" in natural language.

Please break the paragraphs into lines of proper width.

Reply:
Sure, I will break  phrases into proper line width and add an entry in [nsdhw_23au/project/README.rst](https://github.com/yungyuc/nsdhw_23au/blob/fbce131356de2ba33c445eb217941ad35c61f25d/project/README.rst?plain=1#L55)